_blog.yml

```
@@ -3935,3 +3935,15 @@
     - research
     - leaderboard
     - LLM
+    
+- local: energystarai_intro
```

```suggestion
- local: energy-star-ai
```
more consistent name/URL